include/z64bgcheck.h

```diff
@@ -155,10 +155,15 @@ typedef struct {
     /* 0x60 */ f32 maxY;
 } BgActor; // size = 0x64
 
+#define BG_DYNA_ACTOR_FLAG_IS_USED (1 << 0) // The bgActor entry is in use
```

Comment:
```suggestion
#define BG_DYNA_ACTOR_FLAG_IN_USE (1 << 0) // The bgActor entry is in use
```
Past tense to current tense? Maybe even something like `_IS_ACTIVE`? But I think this is fine

Reply:
gotta watch out for confusion with the `_DISABLED` one, to tell apart "bgActor entry is used" and "collision is used ('applied')"

Reply:
no reason `_DISABLED` can't just be `_COLLISION_DISABLED`

Additionally, I don't think it's wholly necessary to have every flag define have `FLAG` in the name

Reply:
wouldn't it be confusion to have both `bg_dyna_actor_is_active` and `bg_dyna_actor_collision_disabled` ? Those seem to contradict each other